examples/generic_connector_configs/README.md

```diff
@@ -646,6 +647,26 @@ The configuration file for the New Relic API can be found at
 
 ___
 
+### OAuth 1.0 API
+
+This generic OAuth HTTP connector can be used for any service that uses OAuth1
+for authorization.
+
+The configuration file for the OAuth 2.0 API can be found at
```

Comment:
```suggestion
The configuration file for connecting to an API that uses OAuth 1.0 can be found at
```

Reply:
do we need to also specify something about the specific type of hashing currently supported?

Reply:
Good point! Will update :). Also thanks for catching my typo!